packages/components/src/components/omnibox/actions/Filter.ts

```diff
@@ -412,12 +414,18 @@ export class FilterAction implements Action {
 
         return strValues.map(strValue => {
             const value = `"${col.shortCaption}" ${operator} ${strValue}`;
+            let displayValue = value;
+            if (this.getFilterDisplayValue) {
+                const altDisplayValue = this.getFilterDisplayValue(col.shortCaption, strValue);
```

Comment:
Do you think we should instead just pass `col` as the first param and then let the usage be the one that decides that they want to use the `col.shortCaption` in there criteria?

Reply:
huh, if col is not always available, wouldn't the call for `col.shortCaption` cause issues?

Reply:
Sorry, I didn't state this clearly. col is available in the function, but not always available elsewhere when getFilterDisplayValue is called. For instance, in actionValueFromFilter function, there is only filter.getColumnName that's available, without the column info.

Reply:
Got it. That makes sense.